script.js

```diff
@@ -1,17 +1,49 @@
-$('.save-btn').on('click', createIdea);
+$('.save-btn').on('click', createCard);
 $('.new-idea-section').on('click', deleteIdea);
+$('.new-idea-section').on('click', upvoteIdea);
+$('.new-idea-section').on('click', downvoteIdea);
 
-function createIdea(event) {
+
+recallCards();
+
+function upvoteIdea(event) {
+  if ($(event.target).siblings('.quality').text() === 'Quality: swill') {
+   $(event.target).siblings('.quality').text('Quality: plausible');
+   var id = $(event.target).parent().attr('id');
+   var retrieveIdea = localStorage.getItem(id);
+   var parsedIdea = JSON.parse(retrieveIdea);
+   console.log(parsedIdea);
+   parsedIdea.quality = 'plausible';
+   var stringifiedIdea = JSON.stringify(parsedIdea);
+   localStorage.setItem(id, stringifiedIdea);
+  }
+
+}
+
+function downvoteIdea() {
+
+}
+function generateCard(id, title, body) {
+  return `<article class="new-idea" id="${id}">
+            <article class="idea-header">
+              <h2 class="idea">${title}</h2>
+              <button class="delete-btn"></button>
+            </article>
+            <p>${body}</p>
+            <button class="upvote-btn"></button>
+            <button class="downvote-btn"></button>
+            <p class="quality">Quality: swill</p>
+          </article>`;
+}
+
+function createCard(event) {
   event.preventDefault();
-  $('.new-idea-card').prepend(`<article class="new-idea">
-      <article class="idea-header">
-        <h2 class="idea">${$('.title-input').val()}</h2>
-        <button class="delete-btn"></button>
-      </article>
-      <p>${$('.idea-input').val()}</p>
-      <button class="upvote-btn"></button>
-      <button class="downvote-btn"></button>
-    </article>`);
+  var titleInput = $('.title-input').val();
+  var ideaInput = $('.idea-input').val();
```

Comment:
I would recommend renaming these vars. the var here represents the actual value and not the input element.